src/lib/channel-manager.js

```diff
@@ -1,23 +1,26 @@
 var SimplePeer = require('simple-peer');
+var wrtc = require('wrtc');
```

Comment:
That's again one of the node-js related changes. 
I currently have it running in nodejs, but obviously a better mechanism is required to make the same library usable in the browser and in nodejs.